### PR TITLE
Bump updates

### PR DESCRIPTION
**Pull request by bitPogo**
### :pushpin: References
* **Issue:** [SDK-440](https://gesundheitscloud.atlassian.net/browse/SDK-440)
* **Related pull-requests:** #1, #2

### :thinking: DOD Checklist

- [x] My code follows the code style and naming conventions of this project.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I have updated the documentation.
- [x] I have updated the changelog accordingly.

### CODE REVIEW

File: gradlePlugin/test-util-dependency/src/main/kotlin/care/data4life/gradle/util/test/dependency/Version.kt [hunk 1]
```diff
@@ -66,7 +65,7 @@ object Version {
     /**
      * [Material Android](https://github.com/material-components/material-components-android)
      */
-    const val material = "1.4.0"
+    const val material = "1.6.0-rc01"
```
Comment (wmontwe) [inline gradlePlugin/test-util-dependency/src/main/kotlin/care/data4life/gradle/util/test/dependency/Version.kt:68]: Material 1.5.0, please

File: test-util-coroutine/build.gradle.kts [hunk 2]
```diff
@@ -39,6 +39,22 @@ kotlin {
     ios {}
 
     sourceSets {
+        removeAll { sourceSet ->
```
Comment (wmontwe) [inline test-util-coroutine/build.gradle.kts:42]: depends on `androidTest`

File: test-util-coroutine/build.gradle.kts [hunk 3]
```diff
@@ -101,6 +117,7 @@ kotlin {
 
 android {
     compileSdk = LibraryConfig.android.compileSdkVersion
+    resourcePrefix =LibraryConfig.android.resourcePrefix + "coroutine_"
```
Comment (wmontwe) [inline test-util-coroutine/build.gradle.kts:120]: spacing

File: test-util-ktor/build.gradle.kts [hunk 4]
```diff
@@ -38,6 +38,15 @@ kotlin {
     ios {}
 
     sourceSets {
+        removeAll { sourceSet ->
+            setOf(
```
Comment (wmontwe) [inline test-util-ktor/build.gradle.kts:42]: depends on `androidTest`

File: test-util-ktor/build.gradle.kts [hunk 5]
```diff
@@ -96,6 +105,7 @@ kotlin {
 
 android {
     compileSdk = LibraryConfig.android.compileSdkVersion
+    resourcePrefix =LibraryConfig.android.resourcePrefix + "ktor_"
```
Comment (wmontwe) [inline test-util-ktor/build.gradle.kts:108]: spacing

File: test-util/build.gradle.kts [hunk 6]
```diff
@@ -39,6 +39,15 @@ kotlin {
     ios {}
 
     sourceSets {
+        removeAll { sourceSet ->
```
Comment (wmontwe) [inline test-util/build.gradle.kts:42]: depends on `androidTest`

**Review by wmontwe — CHANGES_REQUESTED** [review]
Some little findings

File: test-util-ktor/build.gradle.kts (hunk 5 above)
Comment (bitPogo) [reply test-util-ktor/build.gradle.kts:109]: done

File: test-util/build.gradle.kts (hunk 6 above)
Comment (bitPogo) [reply test-util/build.gradle.kts]: done

File: test-util-coroutine/build.gradle.kts (hunk 3 above)
Comment (bitPogo) [reply test-util-coroutine/build.gradle.kts]: done

File: test-util-coroutine/build.gradle.kts (hunk 2 above)
Comment (bitPogo) [reply test-util-coroutine/build.gradle.kts]: done

File: gradlePlugin/test-util-dependency/src/main/kotlin/care/data4life/gradle/util/test/dependency/Version.kt (hunk 1 above)
Comment (bitPogo) [reply gradlePlugin/test-util-dependency/src/main/kotlin/care/data4life/gradle/util/test/dependency/Version.kt:68]: done

**Review by wmontwe — APPROVED** [review]
👍

**Pull request merged**